pkg/sparrow/metrics/exporter.go

```diff
@@ -100,8 +100,10 @@ func newHTTPExporter(ctx context.Context, config *Config) (sdktrace.SpanExporter
 		otlptracehttp.WithEndpoint(config.Url),
 		otlptracehttp.WithHeaders(headers),
 	}
-	if tlsCfg != nil {
-		opts = append(opts, otlptracehttp.WithTLSClientConfig(tlsCfg))
+	if config.Tls.Enabled {
```

Comment:
Please merge into `if config.Tls.Enabled && tlsCfg != nil`. We can avoid a nested if that way, since there actually is no real need for it, without sacrificing readability.

Reply:
Same as [here](https://github.com/caas-team/sparrow/pull/212#discussion_r1804336193)